src/components/SpecialResolution/HelpSpecialResolution.vue

```diff
@@ -0,0 +1,200 @@
+<template>
+  <div>
+    <div class="help-section-component ma-6">
+      <div
+        class="help-btn top"
+        @click="helpToggle = !helpToggle"
+      >
+        <v-icon color="primary">
+          mdi-help-circle-outline
+        </v-icon>
+        <span
+          v-if="!helpToggle"
+          class="pl-2"
+        >Help with Special Resolution</span>
+        <span
+          v-else
+          class="pl-2"
+        >Hide Help</span>
+      </div>
+      <v-expand-transition v-if="helpToggle">
+        <section class="help-section info-text">
+          <header class="help-header mt-4">
+            <h2 id="sr-heading">
+              Help with Special Resolution
+            </h2>
+          </header>
+          <br>
+          <div>
+            <p>
+              A special resolution is a resolution by the members of a Cooperative Association.
+              It is required if you wish to change the Cooperative Name, Type, Rules of Association
+              or Memorandum of Association.
+            </p>
+            <p>
+              A special resolution is submitted to a vote.
+              It is passed by either being consented to in writing by all members, or in a general meeting.
+            </p>
+            <p>
+              For details on how many members need to vote in a general meeting, refer to the
+              <a href="https://www.bclaws.gov.bc.ca/civix/document/id/complete/statreg/99028_01">
+                Cooperative Association Act
+                <v-icon
+                  dense
+                  color="primary"
+                  class="ml-1 mt-n0"
+                >
+                  mdi-open-in-new
+                </v-icon>
+              </a>
+            </p>
+            <p>The special resolution must include:</p>
+            <ul>
+              <li>
+                The name of the Cooperative Association, incorporation number, date the resolution was passed,
+                and the full text of the resolution which clearly indicates the changes being made.
+              </li>
+              <li>
+                The signature of a current director, officer or lawyer of the Cooperative Association and
+                the date of the signature.
+              </li>
+            </ul>
+          </div>
+          <br>
+          <br>
```

Comment:
Not a big deal but if you're in this code again, used Vuetify whitespace classes instead of BRs -- save those for line breaks in paragraphs ;)

Reply:
Fixed, thanks. Though I am surprised we need a 48px whitespace...?